app/javascript/article-form/components/Help/BasicEditor.jsx

```diff
@@ -1,18 +1,21 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
+import { i18next } from '@utilities/locale';
 
 export const BasicEditor = ({ openModal }) => (
   <div
     data-testid="basic-editor-help"
     className="crayons-card crayons-card--secondary p-4 mb-6"
   >
-    You are currently using the basic markdown editor that uses{' '}
+    {i18next.t('editor.help.basic.desc1')}
```

Comment:
This run of text should have been concatenated into one along with some other longer text in the same directory.